qiita_pet/static/js/sharing.js

```diff
@@ -1,4 +1,4 @@
-var current_study = null;
+var current_id = null;
```

Comment:
Can the global variable be removed? Recommend taking a look at the [data](  http://www.w3schools.com/tags/att_global_data.asp) variables. They're pretty useful and they allowed me to remove all the global variables that was in the previous code.